tests/acceptance/features/coreApiVersions/fileVersions.feature

```diff
@@ -480,8 +452,8 @@ Feature: dav-versions
     And user "Alice" has shared file "textfile0.txt" with user "Brian"
     And user "Brian" has accepted share "/textfile0.txt" offered by user "Alice"
     When user "Brian" tries to get versions of file "textfile0.txt" from "Alice"
-    Then the HTTP status code should be "207"
-    And the number of versions should be "3"
+    Then the HTTP status code should be "403"
+    And the number of versions should be "0"
```

Comment:
You can delete this step because it doesn't count versions correct in Brians case